pkg/imgpkg/image/registry.go

```diff
@@ -35,9 +33,10 @@ const (
 )
 
 type RegistryOpts struct {
-	CACertPaths []string
-	VerifyCerts bool
-	Insecure    bool
+	CACertPaths                   []string
+	VerifyCerts                   bool
+	Insecure                      bool
+	IncludeNonDistributableLayers bool
```

Comment:
i would throw in a newline before this since first 3 fields are related to connectivity and this one isnt.

Reply:
ok.